packages/react-charting/src/components/AreaChart/__snapshots__/AreaChartRTL.test.tsx.snap

```diff
@@ -1222,7 +1222,7 @@ exports[`Area chart rendering Should render the area chart with date x-axis data
             d="M40,-31.382C39.024,-27.401,38.049,-23.421,37.073,-23.421C33.171,-23.421,29.268,-30.658,25.366,-30.658C15.122,-30.658,4.878,-13.289,-5.366,-13.289C-10.244,-13.289,-15.122,-20.526,-20,-27.763L-20,-35C-15.122,-35,-10.244,-35,-5.366,-35C4.878,-35,15.122,-35,25.366,-35C29.268,-35,33.171,-35,37.073,-35C38.049,-35,39.024,-35,40,-35Z"
           >
             <circle
-              aria-label="1/6/2018, 12:00:00 AM. legend1, 5."
+              aria-label="6/1/2018, 12:00:00 am. legend1, 5."
```

Comment:
this change looks suspicious

Reply:
Yes @AtishayMsft , informed to kshitij on this and i will work with him on this. Even in other PR also i can see same issue.